depth-estimation/src/types.ts

```diff
@@ -0,0 +1,63 @@
+/**
+ * @license
+ * Copyright 2022 Google LLC. All Rights Reserved.
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ * =============================================================================
+ */
+import {Tensor2D} from '@tensorflow/tfjs-core';
+
+import {PixelInput} from './shared/calculators/interfaces/common_interfaces';
+
+export enum SupportedModels {
+  PortraitDepth = 'PortraitDepth',
+}
+
+/**
+ * Common config to create the depth estimator.
+ *
+ * `minDepth`: The minimum depth value outputted by the estimator.
```

Comment:
nit: outputted -> output. 
Ref: https://english.stackexchange.com/questions/35418/past-tense-of-to-output-output-or-outputted